### PR TITLE
imprv: S3C-3475 add s3 actions in logs

### PR DESCRIPTION
**Pull request by dora-korpar**
Adds fields needed for audit logs

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

**Review by terrydervaux — APPROVED** [review]
LGTM

File: lib/policyEvaluator/utils/actionMaps.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    bucketPutPolicy: 's3:PutBucketPolicy',
+    objectGetLegalHold: 's3:GetObjectLegalHold',
+    objectGetRetention: 's3:GetObjectRetention',
+    objectPutLegalHold: 's3:PutObjectLegalHold',
+    objectPutRetention: 's3:PutObjectRetention',
+}, sharedActionMap);
+
+// action map for all relevant s3 actions
+const actionMapS3 = Object.assign({
+    bucketGetNotification: 's3:GetBucketNotification',
```
Comment (jonathan-gramain) [inline lib/policyEvaluator/utils/actionMaps.js:71]: minor: you could have those two into its own `actionMapBucketNotifications` variable to make it clearer that S3 is an aggregate of multiple action sets.

File: lib/policyEvaluator/RequestContext.js [hunk 2]
```diff
@@ -7,116 +7,28 @@ const parseIp = require('../ipCheck').parseIp;
...
-
-const _actionMapMetadata = {
-    admin: 'metadata:admin',
-    default: 'metadata:bucketd',
-};
-
 function _findAction(service, method) {
     if (service === 's3') {
-        return _actionMap[method];
+        return actionMapRQ[method];
```
Comment (jonathan-gramain) [inline lib/policyEvaluator/RequestContext.js:25]: isn't it `actionMapS3` that you want here?

Comment (dora-korpar) [reply lib/policyEvaluator/RequestContext.js:25]: no, RQ is for the RequestContext file. S3 is used for all the methods in CloudServer. The lists do converge in 7.8

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**